keystore.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+package pb;
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.gostring_all) = true;
+option (gogoproto.equal_all) = true;
+option (gogoproto.verbose_equal_all) = true;
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) =  true;
+option (gogoproto.populate_all) = true;
+option (gogoproto.testgen_all) = true;
+// uncomment to enable bench generation
+// option (gogoproto.benchgen_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+
+// KeystoreAPI provides a keystore management API
+service KeystoreAPI {
+    // KeystoreHas is used to check if we have the key in our store
+    rpc KeystoreHas(KeystoreRequest) returns (KeystoreResponse) { };
+    // KeystoreGet is used to return a key from our store
+    rpc KeystoreGet(KeystoreRequest) returns (KeystoreResponse) { };
```

Comment:
I don't think we need Has, when we have Get and List.

Reply:
These calls are being modelled after the keystore interface in https://github.com/ipfs/go-ipfs-keystore/blob/master/keystore.go and should be kept the same since the `Has` and `List` are used internally by the keystore.